### PR TITLE
Feature/update resource with personal code

### PR DESCRIPTION
**Pull request by clewis1**
Add uvid to database and add unit tests

### CODE REVIEW

**Comment by clewis1** [comment]
https://companieshouse.atlassian.net/browse/IDVA3-1532

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/routers/personalCodeRouter.int.ts
Comment (PhilipGlover) [inline test/routers/personalCodeRouter.int.ts:1]: my only thought is whether we should have tests for the handler for the Personal Code?  I guess just moving some of these tests around.

**Review by PhilipGlover — APPROVED** [review]
- have run code successfully
  - uvid patched into resource
  - when I then added the statement as well, both were retained
```
    "data" : {
        "company_number" : "00006400",
        "psc_appointment_id" : "1kdaTltWeaP1EB70SSD9SLmiK5Y",
        "verification_details" : {
            "uvid" : "321",
            "statements" : [ 
                "INDIVIDUAL_VERIFIED"
            ]
        }
    },
```
- tests pass and concourse analyse PR job green
- as agreed the refactoring of the tests can be done in a separate story

File: test/routers/personalCodeRouter.int.ts
Comment (clewis1) [reply test/routers/personalCodeRouter.int.ts:1]: New story to be created to address this.

**Pull request merged**